TrackletAlgorithm/MatchProcessor.h

```diff
@@ -912,8 +912,9 @@ void MatchCalculator(BXType bx,
                      ap_uint<MC::LUT_matchcut_r_width>& best_delta_r,
                      const AllStubMemory<ASTYPE>* allstub,
                      const AllProjection<APTYPE>& proj,
-                     ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEIDSize> stubid,
-                     FullMatchMemory<FMTYPE> fullmatch[maxFullMatchCopies]
+                     ap_uint<VMStubBase<VMSType>::kVMSIDSize> stubid,
+                     FullMatchMemory<FMTYPE> fullmatch[maxFullMatchCopies],
+		     int istep
```

Comment:
@aryd Didn't notice before, but `istep` is not used anywhere in this function, so I think it can be removed.
```suggestion
                     FullMatchMemory<FMTYPE> fullmatch[maxFullMatchCopies]
```

Reply:
Removed (I keep adding this argument each time I debug this code as I like to print out on which clock step something is calculated to compare with the emulation...)